### PR TITLE
SQL - Insert

### PR DESCRIPTION
**Pull request by kapnobatai137**
- new discussion insight in the insert workout

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| insert-discussion-insight | <ul><li>`Playground` pointing to `https://student.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2F757d2557a971940305ace59e5bce0808%2Fraw%2F32399b94024073c351727c6888a697f9fa448737%2Fsql-insert-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| insert-discussion-insight | <ul><li>`Playground` pointing to `https://student.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2F757d2557a971940305ace59e5bce0808%2Fraw%2F32399b94024073c351727c6888a697f9fa448737%2Fsql-insert-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: sql/dml/insert/insert-discussion-insight.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+| id  | name                   | year | rating |
+|-----|------------------------|------|--------|
+| 1   | No Country For Old Men | 2007 | 9      |
+| ... | ...                    | ...  | ...    |
+
+How would you insert two more records in this table?
+
+It's up to you to choose which movies you'll insert. They can be your favorite movies, the ones you hate, or just a movie that you like.
+
+It's also up to you to choose the rating system. We have *No Country For Old Men* a nine out of ten, but you can use anything **as long as it's a number**.
```
Comment (Stefan-Stojanovic) [inline sql/dml/insert/insert-discussion-insight.md:36]:
```suggestion
It's also up to you to choose the rating system. We gave *No Country For Old Men* a nine out of ten, but you can use anything **as long as it's a number**.
```

**Review by Stefan-Stojanovic — CHANGES_REQUESTED** [review]

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| insert-discussion-insight | <ul><li>`Playground` pointing to `https://learner.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2F757d2557a971940305ace59e5bce0808%2Fraw%2F32399b94024073c351727c6888a697f9fa448737%2Fsql-insert-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| insert-discussion-insight | <ul><li>`Playground` pointing to `https://learner.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2F757d2557a971940305ace59e5bce0808%2Fraw%2F32399b94024073c351727c6888a697f9fa448737%2Fsql-insert-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**